### ISSUE TITLE
Literature guide

### ISSUE THREAD

**Issue by onierstrasz** [slebok/slebok#11]
It would be nice to have a guide to SLE literature. In particular I would like to see an annotated bibliography of survey articles and introductory literature. Would this belong here or in the [BIBSleigh project](http://bibtex.github.io)?

**Comment by grammarware** [comment]
It's a bit of both: BibSLEIGH must provide the technical means of creating (possibly annotated) paper collections (and it was on my list for quite a long time, but now there is enough motivation to implement it), but actually creating guides is obviously a community endeavour that should happen here and involve discussions and to a certain extent community consensus.

**Comment by anyahelene** [comment]
May be unfortunate to duplicated BIBSLEIGH. Can it be integrated in any way? And made easy to contribute to?

Would make sense to have some sort of link with the glossary. Obviously, it would be good if the glossary linked to defining/seminal papers/books. Perhaps linking to *all* papers is too much?

Extra cool if we could navigate the literature through the terms in the glossary. (Based on semantics, not words)

**Comment by anyahelene** [comment]
And of course, navigatable datebase is no replacement for a literature guide. I.e., like Ralf's YA?? (what was it called? he descriped it at the first OOPSLE)

**Comment by grammarware** [comment]
Sure, I didn't mean duplication: one should be able to browse BibSLEIGH and explicitly package the results of their survey in a JSON there, and link it from here by saying that, for example, a bundle "foobar1" is the complete guide to the history of parsing in SLE.

**Comment by grammarware** [comment]
Then, it will be the responsibility of BibSLEIGH to store it and visualise nicely, and the responsibility of SLEBoK to connect the bundle to the actual body of knowledge (courses, terms, topics, ...). Both are separate projects within the same conceptual initiative,,,

**Comment by onierstrasz** [comment]
I imagine a folder "literature" with one file per topic, each containing an annotated list of key papers to read, with pointers to BIBSLEIGH. Would that make sense?

**Comment by anyahelene** [comment]
We need some sort of interconnected system of information where everything is connected to everything, like in a spider web... Sort of like an inter-web... (cheap paper idea!)

**Comment by grammarware** [comment]
Cheap paper ideas should go into `Cheap_papers.md`

**Comment by anyahelene** [comment]
BTW, literature guide is perhaps particularly important to help peeps find early literature. And there's a terminology/taxonomy task in trying to match up old and obscure terms with something today's researchers will recognize. The language gap is terrifyingly large.

**Comment by grammarware** [comment]
Yes, there is a ton of highly interesting materials that is impossible to find with plain text search just because terminilogy has shifted.

**Comment by anyahelene** [comment]
Also, is this distinct from or the same as the "SLE reader"? #12

**Comment by grammarware** [comment]
In my understanding #12 was more about the list of existing and imaginary overviews and surveys ("paper X compared 100 different language workbenches and feature-modelled them"), while #11 is about collections of papers on a particular topic ("read these 5 papers to understand everything about parsing").

**Comment by onierstrasz** [comment]
I discussed this with Tony and it seems to us that they are very close.

**Comment by inkytonik** [comment]
I think my sense of the SLE reader is that it is more like this kind of literature guide than the description in #12. I.e., an overview of the field from the perspective of key literature, including pointers to that literature. The relevant literature should definitely include available survey papers but would also refer to key non-survey papers.

**Comment by grammarware** [comment]
The directory has been created in a6fe57300f2aff420f4103e9bddf71fb0d16f318
